webapp/src/containers/addOrEditSubscriptions/subComponents/filtersPanel.tsx

```diff
@@ -149,6 +167,50 @@ const FiltersPanel = forwardRef<HTMLDivElement, FiltersPanelProps>(({
         }
     }, [assignmentGroupOptions, serviceOptions]);
 
+    useEffect(() => {
+        if (editing && filters.length) {
+            filters.map((filter) => {
+                const payload: SearchFilterItemsParams = {
+                    search: filter.filterValue ?? '',
+                    filter: filter.filterType,
+                };
+
+                if (filter.filterType === SupportedFilters.ASSIGNMENT_GROUP) {
+                    setGetAssignmentGroupPayload(payload);
+                } else {
+                    setGetServicePayload(payload);
+                }
+
+                makeApiRequestWithCompletionStatus(Constants.pluginApiServiceConfigs.getFilterData.apiServiceName, payload);
+                return '';
```

Comment:
need for this empty return ?

Reply:
was getting eslint error `Array.prototype.map() expects a return value from arrow function`